ci/jobs-dev/run_post_hrrr_URSA.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/sh 
+ 
+#SBATCH -o out.post.hrrr
+#SBATCH -e out.post.hrrr
+#SBATCH -J hrrr_test
+#SBATCH -t 00:20:00
+##SBATCH -q debug
+#SBATCH -q batch
+#SBATCH -A ovp
+#SBATCH --exclusive
+#SBATCH -N 2 --ntasks-per-node=24
+
+set -x
+
+# specify computation resource
+export MP_LABELIO=yes
+export threads=1
+export OMP_NUM_THREADS=$threads
+export APRUN="srun"
+
+echo "starting time"
+date
+
+######################################################################
+# Purpose: to run RAP post processing
+######################################################################
+
+# EXPORT list here
+
+
+module purge
+module use $svndir/modulefiles
+module load ursa_$compiler
+module load wgrib2/3.6.0
+module load prod_util/2.1.1
+module load nccmp/1.9.1.0
+module list
+
+msg="Starting hrrr test"
+postmsg "$logfile" "$msg"
+
+export cmp_grib2_grib2=/home/Wen.Meng/bin/cmp_grib2_grib2_new
+export POSTGPEXEC=${svndir}/exec/upp.x
+
+# CALL executable job script here
+
+# specify your running and output directory
+export startdate=2020060118
+export fhr=04
+export DATA=$rundir/hrrr_${startdate}
+
+export NEWDATE=`${NDATE} +${fhr} $startdate`
+
+export YY=`echo ${NEWDATE} | cut -c1-4`
+export MM=`echo ${NEWDATE} | cut -c5-6`
+export DD=`echo ${NEWDATE} | cut -c7-8`
+export HH=`echo ${NEWDATE} | cut -c9-10`
+
+rm -rf $DATA; mkdir -p $DATA
+cd $DATA
+
+cat > itag <<EOF
+&model_inputs
+fileName='$homedir/data_in/hrrr/wrfout_d01_${YY}-${MM}-${DD}_${HH}_00_00'
+IOFORM='netcdf'
+grib='grib2'
+DateStr='${YY}-${MM}-${DD}_${HH}:00:00'
+MODELNAME='RAPR'
+/
+&NAMPGB
+KPO=47,PO=2.,5.,7.,10.,20.,30.,50.,70.,75.,100.,125.,150.,175.,200.,225.,250.,275.,300.,325.,350.,375.,400.,425.,450.,475.,500.,525.,550.,575.,600.,625.,650.,675.,700.,725.,750.,775.,800.,825.,850.,875.,900.,925.,950.,975.,1000.,1013.2
+/
+EOF
+#FMIN
+
+#copy fix data
+cp $homedir/fix/fix_2.3.0/*bin .
+
+#copy xml
+cp ${svndir}/parm/params_grib2_tbl_new params_grib2_tbl_new
+cp ${svndir}/parm/postxconfig-NT-hrrr.txt postxconfig-NT.txt
+cp ${svndir}/fix/rap_micro_lookup.dat eta_micro_lookup.dat
+
+${APRUN} ${POSTGPEXEC} < itag > wrfpost2.out
+
+# operational hrrr post processing generates 3 files
+filelist="WRFTWO.GrbF04 \
+          WRFPRS.GrbF04 \
+          WRFNAT.GrbF04"
+
+for file in $filelist; do
+export filein2=$file
+ls -l ${filein2}
+export err=$?
+
+if [ $err = "0" ] ; then
+
+ # operational hrrr post processing generates 3 files, start with BGDAWP first
+ # use cmp to see if new pgb files are identical to the control one
+ cmp ${filein2} $homedir/data_out_$compiler/hrrr/${filein2}.${machine}
+
+ # if not bit-identical, use cmp_grib2_grib2 to compare each grib record
+ export err1=$?
+ if [ $err1 -eq 0 ] ; then
+  msg="hrrr test: your new post executable generates bit-identical ${filein2} as the trunk"
```

Comment:
@SamuelTrahanNOAA Please replace the string 'the trunk' with 'the develop branch' in all ursa job cards.